File: Violence-Alert-System/Completed Project/req.py
import requests

url = "http://127.0.0.1:8000/detect_violence"

payload = {}
files=[
  ('video',('D:\code\VD\Violence-Alert-System\Completed Project\V_19.mp4',open('D:\\code\\VD\\Violence-Alert-System\\Completed Project\\V_19.mp4','rb'),'application/octet-stream'))
]
headers = {}

response = requests.request("POST", url, headers=headers, data=payload, files=files)

print(response.text)

File: Full-App/Back-End/req.js
var formdata = new FormData();
formdata.append("video", fileInput.files[0], "D:\code\VD\Violence-Alert-System\Completed Project\V_19.mp4");

var requestOptions = {
  method: 'POST',
  body: formdata,
  redirect: 'follow'
};

fetch("http://127.0.0.1:8000/detect_violence", requestOptions)
  .then(response => response.text())
  .then(result => console.log(result))
  .catch(error => console.log('error', error));